src/delivery/controllers/roverControllers.js

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+import { NASA_API_URL } from "../../config/index.js";
+import Exception from "../../utils/Exception.js";
+
+export const getRoverImage = async (req, res, next) => {
+  try {
+    const { userApiKey } = req.body;
+
+    const nasaApiUrl = `${NASA_API_URL}/mars-photos/api/v1/rovers/curiosity/latest_photos?api_key=${userApiKey}`;
+
+    const response = await axios.get(nasaApiUrl);
```

Comment:
https://teams.microsoft.com/l/message/19:734f8f5c4e1d4c2eb403e0d1790fba23@thread.v2/1729174385609?context=%7B%22contextType%22%3A%22chat%22%7D

need to move logic to service and request to repository